package.json

```diff
@@ -40,16 +41,23 @@
   "devDependencies": {
     "@commitlint/cli": "^8.3.5",
     "@commitlint/config-angular": "^8.3.4",
+    "@semantic-release/changelog": "^5.0.1",
+    "@semantic-release/git": "^9.0.0",
+    "@semantic-release/github": "^7.0.7",
     "@types/jasmine": "^3.5.10",
     "@types/node": "^13.13.5",
     "@typescript-eslint/eslint-plugin": "^2.31.0",
     "@typescript-eslint/eslint-plugin-tslint": "^2.31.0",
     "@typescript-eslint/parser": "^2.31.0",
     "awesome-typescript-loader": "^5.2.1",
     "clean-webpack-plugin": "^3.0.0",
+    "commitizen": "^4.1.2",
+    "conventional-changelog-angular": "^5.0.10",
     "conventional-changelog-cli": "^2.0.34",
```

Comment:
Do we still need conventional-changelog-angular and conventional-changelog-cli?

Reply:
I will try to delete those just in case, you gave me a doubt.

Reply:
Removed these two dependencies.
The angular package was used as a preset and this was a mistake because the configuration should be a custom one with commitizen.
Thank you for pointing that out and you should be aware that I change the Angular preset (default) by the config option using commitizen.
This should work but once again, the dry run won't let me go further enough to test that so a dry run on the master branch will tell us if it's ok for sure.